avaframe/com2AB/com2AB.py

```diff
@@ -322,23 +340,34 @@ def calcAB(AvaProfile, eqParameters):
 
     s = AvaProfile['s']
     z = AvaProfile['z']
+    distance = s[1] - s[0]
+    delta_ind = int(np.floor(30/distance))

```

Comment:
int(np.floor(30/distance))  gives 0 if distance > 30 

Use np.ceil instead?

Reply:
So if distance = 40 I still check the next point?

Reply:
yes i would say so